pkg/api/unversioned/generated.proto

```diff
@@ -21,6 +21,7 @@ syntax = 'proto2';
 
 package k8s.io.kubernetes.pkg.api.unversioned;
 
+import "k8s.io/kubernetes/pkg/runtime/generated.proto";
```

Comment:
Hmm - I don't understand.
The rest of the file didn't change - so why did the imports change?